### DISCUSSION TITLE
Adding a new Footer at the bottom of the page.

### DISCUSSION THREAD

**Discussion by sarinsanyal** [Amna-Hassan04/Serenity-Guide#108] [Ideas]
At the footer, I would like to implement any of some important links which have been listed below:

**1) Copyright Information:**
A copyright notice with the current year and the company's or website's name.
Example: `© 2024 SereniFi. All rights reserved `.

**2) Privacy Policy and Terms of Service:**
Links to the privacy policy and terms of service.

**3) Contact Information:**
 Email address, phone number, or contact form link. 

**4) Social Media Links:**
Icons linking to the site's social media accounts (e.g., Facebook, Twitter, Instagram, LinkedIn).

**5) Newsletter Signup:**
An input field for users to subscribe to a newsletter or updates.

**6) Sitemap:**
A link to the sitemap or a simplified navigation menu to help users find key sections of the site.

**7) Quick Links:**
Links to important pages like About Us, Careers, Blog, or FAQ.

**8) Credits/Attribution:**
Sometimes, websites credit developers, designers, or CMS platforms used to build the site.

**9) Legal Disclaimers:**
Any legal disclaimers or notices relevant to your business or industry.

**10) Accessibility Statement:**
Information about the website's accessibility features.

I would implement it in here:
![image](https://github.com/user-attachments/assets/f0ddb2e6-43ed-4836-8f7d-d1e50faed4d7)

**Comment by Amna-Hassan04** [comment]
@sarinsanyal hi maybe dont add the newsletter part in the footer since we dont have any newsletter for this project.

**Comment by sarinsanyal** [comment]
@Amna-Hassan04 I just put it in there in case you wish to implement a newsletter service and connect it with the backend in the future. 

But, never mind. I am commenting the code out. If you wish to do that in the future, you can always de-comment the code.
